### DISCUSSION TITLE
Print button from MilkdownEditor

### DISCUSSION THREAD

**Discussion by sytolk** [Milkdown/milkdown#755] [Ideas]
Print MilkdownEditor content miss me. I know that web browser have File -> Print but it will not print Milkdown Editor content only

**Comment by Saul-Mirone** [comment]
<img width="1512" alt="Screen Shot 2022-10-10 at 11 16 39" src="https://user-images.githubusercontent.com/10047788/194794717-cab206fa-2cb2-4453-b61f-7d67e2d14c6f.png">
May related to your css. It works fine on our website.

**Reply by sytolk** [reply]
The same page https://milkdown.dev/why-milkdown with the Chrome browser File->Print looks like: 
![C324F181-0957-490B-909E-86E37ED8CF82](https://user-images.githubusercontent.com/1415463/194805055-2522adde-8738-4e0f-b8f3-78baa1e2b61a.png)
My idea was to add in [plugin-menu](https://www.npmjs.com/package/@milkdown/plugin-menu) Print button using something like this: https://github.com/gregnb/react-to-print

**Reply by Saul-Mirone** [reply]
It's css problem, fold the sidebar and the print will work.

**Comment by sytolk** [comment]
yes after closing sidebar print looks the same like your screenshot. but it not perfect :) It will not print only the milkdown content -its have extra menu on the top (with closing sideBar button and so on). How about adding in milkdown plugin-menu Print button?

**Reply by Saul-Mirone** [reply]
You should use query media to control what you want to print in your app. It's not always the same between different apps.
https://developer.mozilla.org/en-US/docs/Web/Guide/Printing
